examples/feature_selection/plot_select_from_model_diabetes.py

```diff
@@ -0,0 +1,98 @@
+"""
+===================================================
+Feature selection using SelectFromModel and LassoCV
+===================================================
+
+Use SelectFromModel meta-transformer along with Lasso to select the best
+couple of features from the diabetes dataset.
+
+Since the L1 norm promotes sparsity of features we might be interested in
+selecting only a subset of the most interesting features from the dataset. This
+example shows how to select two the most interesting features from the diabetes
+dataset.
+
+Diabetes dataset consists of 10 variables (features) collected from 442
+diabetes patients. This example shows how to use SelectFromModel and LassoCv to
+find the best two features predicting disease progression after one year from
+the baseline.
+
+Authors: Manoj Kumar <mks542@nyu.edu>
+         Maria Telenczuk
```

Comment:
you could also point to your github user account page if you want, instead of email. But leaving it as is is also fine.